gui/tasks/forms.py

```diff
@@ -51,6 +51,10 @@ class Meta:
         }
 
     def __init__(self, *args, **kwargs):
+        if "instance" in kwargs:
+            kwargs.setdefault("initial", {})
+            kwargs["initial"]["encryption_password"] = notifier().pwenc_decrypt(kwargs["instance"].encryption_password)
+            kwargs["initial"]["encryption_salt"] = notifier().pwenc_decrypt(kwargs["instance"].encryption_salt)
```

Comment:
pwenc_decrypt may raise an exception here making it impossible to change a cloudsync if it ever happens. Should we handle that exception?